docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1143,6 +1165,83 @@ See the :ref:`hive_examples` for more information.
       Requires ORC format. This property may be shown as true for insert-only
       tables created using older versions of Hive.
     -
+  * - ``partition_projection_enabled``
+    - Enables partition projection for selected table.
+      Mapped from AWS Athena table property
+      `projection.enabled <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-setting-up.html>`_.
+    -
```

Comment:
is there a default somehow for each table?

Reply:
From looking at the code, the default value is null, but that null behaves identically to false because it's read in as an Optional and then used in some checks with `<property>.orElse(false)`. So you do _need_ to set this to true to enable it for a table. Should we say it's default false in the docs (because for our users, it may as well be), then submit a follow-up PR to explicitly set a default of false and get rid of the Optional?

Reply:
It's a little more complex even. Value is optional as we are not owners of it. It may be pre existing in metastore.
So we read it as optional and:
- if Optional is empty - Projection is not active. Thats why in condition there is `<property>.orElse(false)`
- if Optional is not empty
   - if value is false - Projection is there but disabled. We only validate all Projection table and column Properties. If there is some issue users may add Trino owned property `partition_projection_ignore=true` to by pass compatibility issues
   - if value is true - Projection is enabled. We validate all Projection table and column Properties and use Partition Projection
   
All above is due to fact that when in AWS Athena `projection.enabled=false` other properties are not validated and may contain invalid values.
And as we map them to Trino properties and the other way round we may run into issues. 
So we prefere to always validate if `projection.enabled` is there. Or tell Trino to ignore that table projection at all.

Reply:
And so the default is `null`